db/migrate/20230914064811_add_unique_index_on_blob_checksum.rb

```diff
@@ -26,8 +26,8 @@ def change
         UPDATE storage_tables_blobs
         SET attachments_count = attachments_count - 1,
             attachments_count_modified = CURRENT_TIMESTAMP
-        WHERE checksum = NEW.checksum;
-        RETURN NEW;
+        WHERE partition_key = OLD.blob_key AND checksum = OLD.checksum;
+        RETURN OLD;
```

Comment:
In een AFTER trigger maakt het niet uit wat je returned, wordt niks mee gedaan. Ik vind het zelf duidelijk om NULL te returnen als de function alleen als AFTER aangeroepen mag worden. Want als je deze toch in BEFORE aanroept gaat het stuk. Anders is het logisch om NEW te returnen zodat het niet stuk gaat vanuit een BEFORE. Het returnen van OLD komt niet vaak voor, volgens mij alleen als je update wilt terugdraaien vanuit een trigger.

```suggestion
        RETURN NULL;
```